build/README.md

```diff
@@ -229,9 +233,12 @@ Now that the images are pushed, to install the development version (with all ima
 run `make install` and Agones will install the image that you just built and pushed on the test cluster you
 created at the beginning of this section. (if you want to see the resulting installation yaml, you can find it in `build/.install.yaml`)
 
-Finally to run end-to-end tests against your development version previously installed in your test cluster run `make test-e2e`, this will validate the whole application flow (from start to finish). If you're curious about how they work head to [tests/e2e](../test/e2e/)
+Finally, to run all the end-to-end tests against your development version previously installed in your test cluster run 
+`make test-e2e` (this can also take a while), this will validate the whole application flow (from start to finish). If 
```

Comment:
nit: Period after the parenthetical statement and start "This will validate" as a new sentence.